maas/plugins/nova_cloud_stats.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+
+# Copyright 2014, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import collections
+
+import ipaddr
+from maas_common import get_auth_ref
+from maas_common import get_nova_client
+from maas_common import metric
+from maas_common import print_output
+from maas_common import status_err
+from maas_common import status_ok
+
+# The actual stat names from novaclient are nasty, so this mapping is used to
+# translate them to something more consistent and usable, as well as set the
+# units for each metric
+stats_mapping = {
+    'hypervisor_count': {
+        'stat_name': 'count', 'unit': 'hypervisors', 'type': 'uint32'
+    },
+    'total_disk_space': {
+        'stat_name': 'local_gb', 'unit': 'gb', 'type': 'uint32'
```

Comment:
I think the units should be of the appropriate case e.g. GB not gb. On this point, in other places the words have been used instead. It would be nice to standardise.